detection-rules/impersonation_recipient_sld_in_sender_local_fts.yml

```diff
@@ -5,7 +5,13 @@ description: |
   the sender's domain is not a known org domain, and it's a first time sender.
 source: |
   type.inbound
-  and any(recipients.to, strings.contains(sender.email.local_part, .email.domain.sld))
+  and (
+      any(recipients.to, 
+          (strings.contains(sender.email.local_part, .email.domain.sld)
+          // checking to ensure no FPs, like "me" in "me.com"
+          and length(.email.domain.sld) > 3)
+      )
```

Comment:
nit:
```suggestion
          strings.contains(sender.email.local_part, .email.domain.sld)
          // checking to ensure no FPs, like "me" in "me.com"
          and length(.email.domain.sld) > 3
      )
```

Reply:
this was intentional :p
MQL evaluates the same `.email.domain.sld` with the parens, and doesn't without them.